app/upgrades/v20/upgrades.go

```diff
@@ -25,7 +33,53 @@ func CreateUpgradeHandler(
 			return vm, err
 		}
 
+		InitializeMaxProviderConsensusParam(ctx, keepers.ProviderKeeper)
+		InitializeMaxValidatorsForExistingConsumers(ctx, keepers.ProviderKeeper)
+		SetMaxValidatorsTo200(ctx, *keepers.StakingKeeper)
+
 		ctx.Logger().Info("Upgrade v20 complete")
 		return vm, nil
 	}
 }
+
+// InitializeMaxValidatorsForExistingConsumers initializes the max validators
+// parameter for existing consumers to the MaxProviderConsensusValidators parameter.
+// This is necessary to avoid those consumer chains having an excessive amount of validators.
+func InitializeMaxValidatorsForExistingConsumers(ctx sdk.Context, providerKeeper providerkeeper.Keeper) {
+	maxVals := providerKeeper.GetParams(ctx).MaxProviderConsensusValidators
+	for _, chainID := range providerKeeper.GetAllRegisteredConsumerChainIDs(ctx) {
+		providerKeeper.SetValidatorSetCap(ctx, chainID, uint32(maxVals))
```

Comment:
proposed chains would not have their validator cap set to 180.
but all chains we have on the horizon are opt-in chains that specify a validator set cap, anyways.
if this is not ultimately the case, then we need to adjust this.